File: NirvanaSMS.Api.Java/src/main/java/org/nirvanasms/api/core/interfaces/IMessageSender.java
package org.nirvanasms.api.core.interfaces;

import org.nirvanasms.api.core.base.MessageBase;
import org.nirvanasms.api.core.entity.ApiUser;
import org.nirvanasms.api.core.utility.ProcessResult;

public interface IMessageSender {
	ProcessResult<String> sendMessage(ApiUser apiUser, MessageBase message);
}

File: NirvanaSMS.Api.CSharp/LibraryProject/Core/Interface/IMessageSender.cs
using NirvanaSMS.Api.Core.Base;
using NirvanaSMS.Api.Core.Entity;
using NirvanaSMS.Api.Core.Utility;

namespace NirvanaSMS.Api.Core.Interface
{
    interface IMessageSender
    {
        ProcessResult<string> SendMessage(ApiUser apiUser, MessageBase message);
    }
}
